pkg/route/api/validation/validation.go

```diff
@@ -32,6 +32,11 @@ func ValidateRoute(route *routeapi.Route) fielderrors.ValidationErrorList {
 		result = append(result, fielderrors.NewFieldInvalid("path", route.Spec.Path, "path must begin with /"))
 	}
 
+	if len(route.Spec.Path) > 0 && route.Spec.TLS != nil &&
+		route.Spec.TLS.Termination == routeapi.TLSTerminationPassthrough {
```

Comment:
@liggitt Not sure if I understood you correctly, there should be no changes vis-a-vis the fields - they should be converted from the old to the new schema. 

If you mean for passthrough routes with paths, well those would have been redundant anyway - would have never worked ("cruft" in the datastore / etcd) as they would have just generated unused config. So this is actually indicating that those routes are invalid and it would log errors (and wouldn't try to add a route to the router). 

And re: route status update - no, this is not called in the path via `ValidateRouteStatusUpdate`, so it would not affect status update.